jobs/service-fabrik-broker/templates/config/keepalived/notify.sh.erb

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# notify.sh
+
+echo $3 > /var/run/keepalive.$1.$2.state
```

Comment:
can we use descriptive var names here, and use them? for eg:
```
TYPE=$1
NAME=$2
STATE=$3
```

Reply:
sure . Would update.